lib/rest-api.php

```diff
@@ -18,3 +18,29 @@ function gutenberg_register_global_styles_endpoints() {
 	$global_styles_controller->register_routes();
 }
 add_action( 'rest_api_init', 'gutenberg_register_global_styles_endpoints' );
+
+/**
+ * Adds the post classes to the REST API response.
+ *
+ * @param WP_REST_Response $response    Response object.
+ * @param WP_Post          $post    Post object.
+ *
+ * @return WP_REST_Response Response object.
+ */
+function gutenberg_add_post_class_to_api_response( $response, $post ) {
+	$response->data['post_class'] = get_post_class( '', $post->ID );
```

Comment:
We don't prefix things in the posts API with `post`. I'd name this `class_list`.